frameworks/Dart/aqueduct/benchmark_config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "framework": "aqueduct",
+  "tests": [{
+    "default": {
+      "json_url": "/json",
+      "plaintext_url": "/plaintext",
+      "db_url": "/db",
+      "query_url": "/queries/",
+      "update_url": "/updates/",
+      "fortune_url": "/fortunes",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Micro",
```

Comment:
What is the distinction for 'Micro'?

Reply:
Unsure of exact requirements, but Redstone is marked Micro also, so felt appropriate.